packages/cli-kit/src/store.ts

```diff
@@ -44,21 +42,13 @@ let _instance: CLIKitStore | undefined
 
 export function cliKitStore() {
   if (!_instance) {
-    throw new Abort("The CLIKitStore instance hasn't been initialized")
-  }
-  return _instance
-}
-
-export async function initializeCliKitStore() {
-  if (!_instance) {
-    // eslint-disable-next-line require-atomic-updates
     _instance = new CLIKitStore({
       schema,
       migrations,
       projectName: 'shopify-cli-kit',
-      projectVersion: await constants.versions.cliKit(),
```

Comment:
I removed the projectVersion as the [default value](https://github.com/sindresorhus/conf/tree/v10.1.2#projectversion) works for us (the version from the closest package.json). This way, there's no need to make the method async.